contracts/storage/RibbonTreasuryVaultStorage.sol

```diff
@@ -2,83 +2,29 @@
 pragma solidity =0.8.4;
 
 import {Vault} from "../libraries/Vault.sol";
```

Comment:
Similar to [here](https://github.com/ribbon-finance/ribbon-v2/pull/311/files#r1180634246)

Reply:
can you elaborate?

Reply:
right, what I meant is that this import `import {Vault} from "../libraries/Vault.sol";` may no longer be needed given the changes on the file